.gitignore

```diff
@@ -16,6 +24,14 @@ HasalServer.log.*
 stage/*
 docs/*
 
+# File types to ignore
+*.zip
+*.diff
+*.json
```

Comment:
There are some JSON configuration files in Hasal, I think we shouldn't ignore JSON file.